blocks/header/header.css

```diff
@@ -324,3 +324,309 @@ header .breadcrumbs ol li[aria-current] {
     padding: 0 2rem;
   }
 }
+
+/* !! new styles start here */
+
+.nav-toolbar {
+  display: flex;
+  flex-direction: row;
+  gap: 1.5rem;
+  align-items: center;
+}
+
+.theme-toggle {
+  --spectrum-component-size-minimum-perspective-down: 1.5rem;
```

Comment:
these CSSCPs come from the spectrum repo, I left them here for integration reasons, but if it's cumbersome I'll refactor them out.